redisinsight/ui/src/pages/rdi/pipeline-management/PipelineManagementPage.tsx

```diff
@@ -34,7 +35,10 @@ const PipelineManagementPage = ({ routes = [] }: Props) => {
   setTitle(`${rdiInstanceName} - Pipeline Management`)
 
   useEffect(() => {
-    dispatch(fetchRdiPipeline(rdiInstanceId))
+    if (!pipelineData) {
```

Comment:
Maybe will better move dispatch(fetchRdiPipeline(rdiInstanceId)) to RdiInstancePage ?